misc/userscripts/qutedmenu

```diff
@@ -50,4 +50,6 @@ url=${url/*http/http}
 case $1 in
     open) printf '%s' "open $url" >> "$QUTE_FIFO" || qutebrowser "$url" ;;
     tab)    printf '%s' "open -t $url" >> "$QUTE_FIFO" || qutebrowser "$url"  ;;
+    window)    printf '%s' "open -w $url" >> "$QUTE_FIFO" || qutebrowser "$url"  ;;
+    private)    printf '%s' "open -p $url" >> "$QUTE_FIFO" || qutebrowser "$url"  ;;
```

Comment:
```suggestion
    private)    printf '%s' "open -p $url" >> "$QUTE_FIFO" || qutebrowser --target private-window "$url"  ;;
```

otherwise, the URL gets opened in non-private mode if the `printf` failed, which would not be what the user expects.